src/app/public/modules/validators/validators.ts

```diff
@@ -12,6 +12,13 @@ import {
 // @dynamic
 export class SkyValidators {
 
+  /**
+   * Validates email addresses in reactive forms. Add this validator directly to the form control
```

Comment:
@Blackbaud-SteveBrush and @Blackbaud-AlexKingman, I added this JSDoc comment and the one below for the URL function because they are included in the current docs at https://developer.blackbaud.com/skyux/components/email-validation#email-validator-function-properties and https://developer.blackbaud.com/skyux/components/url-validation#url-validation-input-directive-properties. However, `docs-tools` is not including this content on the Development tab. Do we need a `docs-tools` issue? Or am I doing something wrong with the JSDocs comment? Or is this maybe stuff that shouldn't be in the docs after all?

Reply:
Related: https://github.com/blackbaud/skyux-docs-tools/issues/109

Reply:
I'll resolve this issue under the assumption that this will be fixed when we address that `docs-tools` issue and update the `docs-tools` version in this repo. Just let me know if that's not the case, @Blackbaud-SteveBrush.